### PR TITLE
Allows adding link to the citation in the epigraph

### PR DESCRIPTION
**Pull request by dwarkeshsp**
Hey there! I made a simple addition to epigraph.html such that the following shortcode becomes valid:

`{{< epigraph pre="Byrne Hobart," cite="Foxes and Hedgehogs" link="https://byrnehobart.medium.com/foxes-and-hedgehogs-606d6e038b3e">}}`

It basically wraps the `cite` in an `<a>` tag with the `href` being the link value.

### CODE REVIEW

File: layouts/partials/brand.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <header class="brand">
-  <h1>{{ .Site.Title}}</h1>
+  <a href="/"><h1>{{ .Site.Title}}</h1></a>
```
Comment (slashformotion) [inline layouts/partials/brand.html:2]: remplace `href="/"` by `href="{{ .Site.BaseURL }}"`

**Review by slashformotion — COMMENTED** [review]
You can't put "/" as the href. If the website is hosted in a subfolder (ex: `www.domain.com/myawesomeblog/` ). the link `<a href="/"></a>` will redirect to `www.domain.com/`. Which is of course something not supposed to happen. Please change this. 

Thanks for your contribution !

File: layouts/partials/brand.html (hunk 1 above)
Comment (dwarkeshsp) [reply layouts/partials/brand.html]: Done!

Comment (dwarkeshsp) [reply layouts/partials/brand.html]: You don't have to merge the link title if you don't want. I was only doing it for my site, but it ended up in the same pull request.

**Pull request merged**